### DISCUSSION TITLE
Selected tab goes transparent when window state is inactive

### DISCUSSION THREAD

**Discussion by Asad-13** [black7375/Firefox-UI-Fix#882] [Q&A] [answered]
When unfocusing/inactivating the window (focusing on another window), the selected tab colour goes from white/matching with toolbar colour to slightly transparent.

Here is it demonstrated with the default settings:

Active state:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/87184327/5e1f9521-70a8-4040-b429-d58127c5b505)

Inactive state:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/87184327/2129a2a4-46ab-470f-bdec-db9243dfa688)

This is more apparent when using the [Photon Colours theme](https://addons.mozilla.org/en-US/firefox/addon/photon-colors) -

Active state:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/87184327/0c190be3-4752-43f1-9cb9-faa1b7201e1c)

Inactive state:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/87184327/9e9deb7b-2098-43c8-9ea2-55c0438f652e)

Notice how the selected tab goes transparent? I don't want that to happen, such that, even in an inactive state, the selected tab colour is the same as in an active state (i.e. in this case, same as the toolbar colour). Is there a way that I can achieve this?

**Comment by black7375** [comment, marked as answer]
This appears to be a bug with browser version updates, not intended.
I'll be fix it.

**Reply by Asad-13** [reply]
Thanks; I would like to ask another question. Is there a way to modify the colours of the tab bar to match the aforementioned Proton Colours theme, while using the auto-light/dark mode - which is built into Firefox? I love those Proton colours, but I also love my dark mode, so kind of in a dilemma here 🤔

My ideal case scenario -
For light mode, it follows the proton-colour theme (either through directly selecting proton colour theme or by adjusting userChrome values, if possible)
For dark mode, it follows the default dark mode colours (firefox's or your css-tweaked, whichever is used by default when applying this css theme)

**Reply by black7375** [reply]
I saw late to see if there was an answer.
It is possible to completely turn off system theme options, but mixing them is a tricky subject.

ㅑ recommend that you refer to [this file](https://github.com/black7375/Firefox-UI-Fix/blob/master/src/theme/system_default_theme/_win10.scss) and add modifications to `userChrome.css`.
```css
@media (-moz-bool-pref: "userChrome.theme.system_default") and (-moz-platform: windows) {
  :root:is(:not([lwtheme]), :not(:-moz-lwtheme)) {
     --win-text-color: red;
  }
}
```
